docs/advanced/solo-proof-of-stake-voting.md

```diff
@@ -0,0 +1,198 @@
+
+---
+
+## Ticket Buying
+Now from your cool wallet you can purchase tickets using the following command once your wallet is unlocked:
+
+`dcrctl --wallet purchaseticket default 150 1 DsHotWalletAddressFromVPS 10`
+
+Replacing `DsHotWalletAddressFromVPS` with your hot wallet voting address which you will generate when you set up your VPS for voting and using the command above will attempt to purchase `10` tickets with a max price of `150` DCR each and delegate voting rights to your VPS instances.
+
+If you wish to automate ticket purchases using ticketbuyer you will need to add the following info in your `~/.dcrwallet/dcrwallet.conf` where `DsHotWalletAddressFromVPS` is once again the address you generated on your voting VPS.
+
+```
+enableticketbuyer=1
+ticketbuyer.votingaddress=DsHotWalletAddressFromVPS
+ticketbuyer.balancetomaintainabsolute=0
+```
+
+---
+
+## Voting
+Setting vote choices for on-chain votes will happen on the VPSs you will set up and we will address that process in a later section. For Politeia votes you will need to cast those from your cool wallet. I will outline the process for doing so using the `politeiavoter` CLI tool.
+
+1. Download the Politeia archive, manifest, and signature files.
+
+`wget https://github.com/decred/decred-binaries/releases/download/v1.3.1/{politeiavoter-linux-amd64-v1.3.1.tar.gz,manifest-politeiavoter-v1.3.1.txt,manifest-politeiavoter-v1.3.1.txt.asc}`
+
+2. Verify the manifest.
+
+`gpg --verify manifest-politeiavoter-v1.3.1.txt.asc`
+
+4. Verify the SHA-256 hash in the manifest matches that of the archive.
+
+`sha256sum politeiavoter-linux-amd64-v1.3.1.tar.gz`
+
+5. Extract the archive.
+
+`tar -xf politeiavoter-linux-amd64-v1.3.1.tar.gz`
+
+6. Enter the `politeiavoter-linux-amd64-v1.3.1` directory.
+
+
+Now to view the various agendas and vote on them you will need to run the following commands. Also remember that for this to work `dcrd` and `dcrwallet` must also be running.
+
+To get a list of active proposals:
+
+`./politeiavoter inventory`
+
+To vote on a proposal:
+`./politeiavoter vote <prop-hash> <yes|no>`
+
+---
+
+# Setting Up "Hot" Voting Wallets
+## VPS Providers
+The first thing you will need is to choose at least one VPS provider. You will find a short list of decent providers below, you may mix and match any number of providers, ideally your VPS should have at least 2 GB of RAM.
+
+* [Scaleway](https://www.scaleway.com/)
+* [Google Cloud Platform (GCP)](https://cloud.google.com/)
+* [Amazon Web Services (AWS)](https://aws.amazon.com/)
+* [OVH](https://www.ovh.com/)
+
+Note: Another consideration is storage space. As you will need to store the complete Decred blockchain which is constantly growing, it is important to keep in mind that while it is currently quite small (~3 GB), it will eventually grow to the point where you may need to upgrade the storage space on your VPS instances.
```

Comment:
Instead of listing 3GB here (which will go out of date) we can link to dcrdata which will have always have an up-to-date value.

We do something similar in the "Blockchain Download" section here http://localhost:8000/wallets/decrediton/decrediton-setup/